.gitignore

```diff
@@ -58,7 +58,7 @@ resources/proto/machine_learning_inference/generated/inference/grpc/generated
 docs/site/
 docs/antora-aissemble-ui/node_modules/
 docs/antora-aissemble-ui/public/
-/extensions/extensions-metadata-service/metastore_db/**
+/extensions/extensions-metadata-service/data/metastore_db/**
```

Comment:
I: moved to `data` directory by default, as that's the writable space in the docker image we package this service in